src/main/java/com/hubspot/jinjava/objects/serialization/PyishObjectMapper.java

```diff
@@ -51,12 +50,7 @@ public static String getAsPyishStringOrThrow(Object val)
     throws JsonProcessingException {
     String string = PYISH_OBJECT_WRITER.writeValueAsString(val);
     Optional<JinjavaInterpreter> interpreterMaybe = JinjavaInterpreter.getCurrentMaybe();
-    Optional<Long> maxStringLength = interpreterMaybe
-      .map(interpreter -> interpreter.getConfig().getMaxStringLength())
```

Comment:
Switched from `getMaxStringLength` to `getMaxOutputSize` because the max string length is used for truncating in the join filter, rather than throwing an output too big exception